src/client/ua_client.c

```diff
@@ -419,6 +419,38 @@ __UA_Client_AsyncService(UA_Client *client, const void *request,
     return UA_STATUSCODE_GOOD;
 }
 
+static UA_StatusCode
+UA_Client_backgroundConnectivityCheck(UA_Client *client) {
+    if(client->config.connectivityCheckInterval) {
```

Comment:
```c
if(!client->config.connectivityCheckInterval)
    return UA_STATUSCODE_GOOD;
```
This reduces the indendation of the remaining method and makes it more readable.